### ISSUE TITLE
Crash report: `TypeError: Cannot read properties of undefined (reading 'includes')`

### ISSUE THREAD

**Issue by flechemano** [KyberNetwork/kyberswap-interface#1297]
## URL

https://kyberswap.com/swap

## Error

```
TypeError: Cannot read properties of undefined (reading 'includes')
```

## Stacktrace

```
TypeError: Cannot read properties of undefined (reading 'includes')
    at rj (https://kyberswap.com/static/js/main.490992f8.chunk.js:1:5379513)
    at Oa (https://kyberswap.com/static/js/main.490992f8.chunk.js:1:2830876)
    at Ts (https://kyberswap.com/static/js/main.490992f8.chunk.js:1:2842564)
    at xc (https://kyberswap.com/static/js/main.490992f8.chunk.js:1:2888999)
    at gu (https://kyberswap.com/static/js/main.490992f8.chunk.js:1:2877052)
    at yu (https://kyberswap.com/static/js/main.490992f8.chunk.js:1:2876980)
    at bu (https://kyberswap.com/static/js/main.490992f8.chunk.js:1:2876843)
    at ou (https://kyberswap.com/static/js/main.490992f8.chunk.js:1:2873623)
    at cu (https://kyberswap.com/static/js/main.490992f8.chunk.js:1:2874014)
    at qi (https://kyberswap.com/static/js/main.490992f8.chunk.js:1:2814597)
    at https://kyberswap.com/static/js/main.490992f8.chunk.js:1:2871556
```

## Device data

```json
{
  "ua": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36",
  "browser": {
    "name": "Chrome",
    "version": "103.0.0.0",
    "major": "103"
  },
  "engine": {
    "name": "Blink",
    "version": "103.0.0.0"
  },
  "os": {
    "name": "Windows",
    "version": "7"
  },
  "device": {},
  "cpu": {
    "architecture": "amd64"
  }
}
```

**Comment by viet-nv** [comment]
Thanks for reporting this issue. We are investing and fixing it ASAP

**Comment by viet-nv** [comment]
Fixed in #1299
